demo/demo.py

```diff
@@ -18,10 +20,84 @@ def parse_args():
         help='whether to use rawframes as input')
     parser.add_argument(
         '--device', type=str, default='cuda:0', help='CPU/CUDA device option')
+    parser.add_argument(
+        '--fps', default=30, help='fps value of the output video')
```

Comment:
fps should be None. if the input is video, then probe the input fps.

Reply:
Modify the description to "specify fps value of the output video when using rawframes to generate file"